src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -206,6 +208,8 @@ export class AppointmentsPage extends Component {
         </div>
         <CancelAppointmentModal
           {...cancelInfo}
+          cernerFacilities={this.props.cernerFacilities}
```

Comment:
I think you can add this `cernerFacilities` info in the `getCancelInfo` selector. Would save you a little bit of boilerplate here.

Reply:
Done.